images/imaging.py

```diff
@@ -486,29 +493,49 @@ def render_galaxy_image(
     snapshot_path = sys.argv[1]
     velociraptor_base_name = sys.argv[2]
     output_path = sys.argv[3]
+    ptype = sys.argv[4]
 
-    halo_ids = range(0, 100)
+    if "COLIBRE" in ptype:
+        halo_ids = range(0, 1)
+    else:
+        halo_ids = range(0, 100)
 
     if not os.path.exists(output_path):
         os.mkdir(output_path)
 
     velociraptor_properties = velociraptor_base_name
     velociraptor_groups = velociraptor_base_name.replace("properties", "catalog_groups")
 
-    filenames = {
-        "parttypes_filename": velociraptor_base_name.replace(
-            "properties", "catalog_partypes"
-        ),
-        "particles_filename": velociraptor_base_name.replace(
-            "properties", "catalog_particles"
-        ),
-        "unbound_parttypes_filename": velociraptor_base_name.replace(
-            "properties", "catalog_partypes.unbound"
-        ),
-        "unbound_particles_filename": velociraptor_base_name.replace(
-            "properties", "catalog_particles.unbound"
-        ),
-    }
```

Comment:
Not sure that this is the best way of dealing with this. Can we do a try/except when we try to load and then if it fails go through and replace catalog_parttypes with catalog_partypes?